src/renderer/Components/RecevingPackage/index.jsx

```diff
@@ -57,22 +116,29 @@ function ReceivingPackage() {
           일련번호
         </label>
         <div className="flex justify-center">
-          {Array(6).fill(
-            <NumberInput
-              validateNumber={validateNumber}
-              handleMoveNextInput={handleMoveNextInput}
-            />,
-          )}
+          {Array(SERIAL_NUMBER_LENGTH)
+            .fill()
+            .map((_, index) => (
+              <NumberInput
+                key={index}
+                onKeyDownFunc={() => {
+                  validateNumber(event, index);
```

Comment:
해당 validateNumber 에서 매개변수로 event 만 필요한데 index 까지 넘겨주고 있습니다.
수정 부탁드리겠습니다. 😄
```suggestion
                  validateNumber(event);
```

Reply:
이 부분은 제가 코멘트 남기면서 잘못 남겨진게 섞여들어갔지 싶네요. 😅

Reply:
확인한 뒤 반영하였습니다! 👍